src/server/models/cacheable_queries/campaign.js

```diff
@@ -49,10 +50,34 @@ const dbCustomFields = async id => {
         .where("campaign.id", id)
     )
     .first();
+
   if (firstContact) {
-    return Object.keys(JSON.parse(firstContact.custom_fields));
+    customFields = Object.keys(JSON.parse(firstContact.custom_fields));
   }
-  return [];
+
```

Comment:
is this part necessary?  This is caching the field list to be sent about the contact -- I don't think we need this at all as these are *texter* fields, not contact fields.

Reply:
@crayolakat to answer @schuyler1d's question here, this is necessary because it's grabbing the existing contact custom fields to be merged with the texter fields to be cached on the campaign, allowing them to be used in scripts.

see this line: 
https://github.com/MoveOnOrg/Spoke/pull/1961/files#diff-612db26be6b7f5a0cdd54b604ffa5d5afb3bd8208271fea120755796f1803972R72